data/powers.js

```diff
@@ -0,0 +1,39 @@
+const powers = [
+  {
+    id: 1,
```

Comment:
Your seed data shouldn't have an `id` property to them since these are automatically added by SQL when seeded into your database.